docs/APPARMOR.md

```diff
@@ -0,0 +1,21 @@
+# apparmor
+
+When running Telegraf under apparmor users may see denial messages depending on
+the Telegraf plugins used and the apparmor profile applied. Telegraf does not
+have control over the apparmor profiles used. If users wish to address denials,
+then they must understand the collections made by their choice of Telegraf
+plugins, the denial messages, and the impact of changes to their apparmor
+profiles.
+
+## Example Denial
+
+For example, users might see denial messages such as:
+
+```s
+type=AVC msg=audit(1588901740.036:2457789): apparmor="DENIED" operation="ptrace" profile="docker-default" pid=9030 comm="telegraf" requested_mask="read" denied_mask="read" peer="dovecot"
```

Comment:
I suggest s/dovecot/unconfined/ here since if we see `peer=dovecot`, it means that telegraf tried to ptrace read a confined process and that process' apparmor profile will also need a corresponding `ptrace (readby) peer=docker-default,` rule, which has all kinds of discussion points (how should I change the dovecot profile? Where is the dovecot profile? Do I want all processes running under docker-default to ptrace read dovecot? How do I create a custom profile for docker under telegraf? How do I create a profile from scratch? etc, etc

By using `peer="unconfined"` in the example, you sidestep a lot of that.